charts/metallb/values.yaml

```diff
@@ -37,6 +37,13 @@ prometheus:
   # certificate to be used.
   speakerMetricsTLSSecret: ""
 
+  controllerMetrics:
```

Comment:
The name is misleading, because the metrics here refer to the controllerRuntime metrics of the controller. Looking at it seems a global metric knob that will enable or disable all of them (see for example the controllerMetricsTLSSecret below, which applies to ALL the metrics produced by the metallb `controller` pod)

I'd have two separate values related only to the ports. If you say 
```
Should either be 0 (disabled) or follow
    # the pattern of :<port>
```

Then I think there's no need for the `enabled` flag. The user can just omit the port or we can default to 0